service-commands/scripts/setup-instance.sh

```diff
@@ -150,7 +111,19 @@ case "$service" in
 			# TODO fix install and provisioning for fast
 		fi
 
+		# configure local files: /etc/hosts, /etc/ssh/ssh_config.d/60-audius.conf
+		configure_etc_hosts
```

Comment:
since this bit asks for the user's local password, perhaps we should log a warning or something to give the user a heads-up that it's because we're trying to `sudo` modify `/etc/hosts` and is not malicious.

Reply:
Ah, ty! Added this: 2f4ef4a141a9bbab5c193ce1802e424f810967c5